tests/unit/test_graphql.py

```diff
@@ -1,8 +1,15 @@
+import httpx
 import pytest
+import respx
```

Comment:
Had to introduce respx here for httpx. But has a nice interface, I like it more than responses :+1:

Reply:
Mhmm but now we have two request test libraries in one project. What is the longterm plan?

That also might confuse contributors. Which response library do I use?

Reply:
@max-wittig currently httpx is only an optional dependency when selecting the `[graphql]` extra.

At some point we can switch to httpx completely with a major release, maybe when httpx hits 1.0.0 (but seems like they are already stable). `responses` isn't able to mock httpx which we use in the GraphQL client, so it's needed here.

Using httpx now lets us add an async GraphQL client very easily and avoid these requests in the future (similar to https://github.com/python-gitlab/python-gitlab/issues/1025, which is still the most upvoted issue, but kind of blocked by the requests/httpx support). If we stick to only requests, it would be another big breaking change on the GraphQL side.

So right now, if REST => `responses`, if GraphQL => `respx`. I could document this maybe.